src/main/resources/static/js/fl-agent.js

```
@@ -0,0 +1,582 @@
+    if(toggle){
+      toggle.addEventListener('click', function(e){
+        if(sidebar && sidebar.classList.contains('open')) closeSidebar(); else openSidebar();
+      });
+    }
+    if(closeBtn) closeBtn.addEventListener('click', closeSidebar);
+    if(overlay) overlay.addEventListener('click', closeSidebar);
+
+    // mode buttons
+    function setMode(mode){
+      console.log('[fl-agent] setMode called with mode:', mode);
+      document.querySelectorAll('.fl-agent-mode-btn').forEach(b=>b.classList.remove('active'));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)